rtp/README.md

```diff
@@ -9,39 +9,100 @@ receiving and sending RTP streams.
 
 You have to have installed the following packages on your system:
 
-* FFmpeg 4.x
-* SDL 2
-* PortAudio
+- FFmpeg 4.x
+- SDL 2
+- PortAudio
+
+### Ubuntu:
 
-One-liner for Ubuntu:
 ```bash
 apt install ffmpeg portaudio19-dev libsdl2-dev
 ```
-One-liner for MacOS:
+
+### MacOS:
+
 ```bash
 brew install ffmpeg portaudio sdl2
 ```
 
-Then, install mix dependencies:
+Furthermore, make sure you have `Elixir` and `Erlang` installed on your machine. For installation details, see: https://elixir-lang.org/install.html
 
-```bash
+On Ubuntu, we recommend installation through asdf, see: https://asdf-vm.com/guide/getting-started.html
+
+## Run the demo
+
+To run the demo, clone the membrane_demo repository and checkout to the demo directory:
+
+```
+git clone https://github.com/membraneframework/membrane_demo
+cd membrane_demo/rtp
+```
+
+Then you need to download the dependencies of the mix project:
+
+```
 mix deps.get
 ```
 
-## Run the demo
+Then you may be asked to install `Hex` and then `rebar3`.
+
+In case of installation issues with Hex on Ubuntu, try updating the system packages first by entering the command:
+
+```shell
+sudo apt-get update
+```
 
-* Open a terminal in the project directory
-* Type `mix run --no-halt receive.exs`
-* Open another terminal in the project directory
-* Type: `mix run --no-halt send.exs`
+In case of issue with compilation of ex_libsrtp, enter:
+
+```shell
+mix deps.update bundlex
+```
+
+and then install pkg-config and srtp (MacOS):
```

Comment:
Just a small note - I think we should somehow redesign the structure of this section once we have all the OS tested, so that to have a separate subsection describing the whole running process for each of the OS.

Reply:
If it could be done easily, then why not ;)

Reply:
I'll deal with it!

Reply:
@varsill Example structure added in `camera_to_hls` readme. Let me know what do you thing about it!

Reply:
that "spoiler" section is a really good idea ;) I really like the new structure

Reply:
Great! Then I will adapt all readme to this and then I will ask you for a final review